assets/javascripts/components/Collapsible/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+export const DEFAULT_SELECTOR = '.collapsible';
+
+const DEFAULT_OPTIONS = {
+    activeClass: 'collapsible',
+    suffixActiveClass: '--active'
+}
+
+class Collapsible {
+    /**
+     * Initializer
+     *
+     * @param {Element} elementRef - Link list element containing with each link item inside
+     * @param {Object} options - Passing some parameters to override default options
+     */
+    constructor(elementRef, options) {
+        // Initialise attributes
+        this.options = Object.assign(DEFAULT_OPTIONS, options);
+        this.activeClass = `${this.options.activeClass}${this.options.suffixActiveClass}`;
```

Comment:
What is teh advantage of composing `this.activeClass` manually 🤔 Couldn't this be part of `DEFAULT_OPTIONS`:

```
const DEFAULT_OPTIONS = {
  activeClass: 'collapsible--active',
}
```

Reply:
@olivierobert The benefit of this is we can override active class into the component integrating on.
The idea is `Collapsible` component will be just a component only in `js` to toggle a `collapsible--active` class by default but we don't have styling for this component. So it just only toggle a class.
And now I have `_link-counter.scss` styling for all of things we need including with `.link-counter_item--active` class, then I can override this `.link-counter_item` to the `Collapsible`.
So once we click the element, it will toggle class with `.link-counter_item--active` instead of default `.collapsible--active` class

Reply:
Oh, I get it now. So that makes sense :-)